backend/app/store/service/service.go

```diff
@@ -24,6 +25,7 @@ type DataStore struct {
 	MaxCommentSize int
 	MaxVotes       int
 	TitleExtractor *TitleExtractor
+	Matcher        *Matcher
```

Comment:
I'd suggest more descriptive name for both fields and strict. Maybe BadWordsMatcher. I little bit long, but easy to understand

Reply:
Agree. Just got confused with examples from our discussion.